lib/std/os.zig

```diff
@@ -2305,8 +2308,15 @@ pub fn chdir(dir_path: []const u8) ChangeCurDirError!void {
     if (builtin.os.tag == .wasi) {
         @compileError("chdir is not supported in WASI");
     } else if (builtin.os.tag == .windows) {
-        const dir_path_w = try windows.sliceToPrefixedFileW(dir_path);
-        @compileError("TODO implement chdir for Windows");
+        windows.SetCurrentDirectory(dir_path) catch |err| switch (err) {
+            error.InvalidUtf8 => return error.InvalidUtf8,
+            error.NameTooLong => return error.NameTooLong,
+            error.FileNotFound => return error.FileNotFound,
+            error.NotDir => return error.NotDir,
+            error.NoDevice => return error.FileSystem,
+            error.AccessDenied => return error.AccessDenied,
+            error.Unexpected => return error.Unexpected,
```

Comment:
There is no need to manually remap corresponding error values across error sets. An `else` prong should be enough:

```suggestion
            error.NoDevice => return error.FileSystem,
            else => |e| return e,
```